airflow/www/extensions/init_appbuilder_links.py

```diff
@@ -21,10 +21,19 @@
 def init_appbuilder_links(app):
     """Add links to Docs menu in navbar"""
     appbuilder = app.appbuilder
-    appbuilder.add_link("Documentation", href=get_docs_url(), category="Docs")
-    appbuilder.add_link("Airflow Website", href='https://airflow.apache.org', category="Docs")
-    appbuilder.add_link("GitHub Repo", href='https://github.com/apache/airflow', category="Docs")
+    appbuilder.add_link(name="docs_links", label="Documentation", href=get_docs_url(), category="Docs")
     appbuilder.add_link(
-        "REST API Reference (Swagger UI)", href='/api/v1./api/v1_swagger_ui_index', category="Docs"
+        name="docs_links", label="Airflow Website", href='https://airflow.apache.org', category="Docs"
+    )
+    appbuilder.add_link(
+        name="docs_links", label="GitHub Repo", href='https://github.com/apache/airflow', category="Docs"
```

Comment:
Yeah this does not feel right

Reply:
cc @jhtimmins any ideas for a better solution?

Reply:
Having a same name for links seems like a hack.

Can we not group them and have a single permission for it. Wdyt @jhtimmins

Reply:
@kaxil I mean, that's basically what this is. There isn't really another way to group these together.